### ISSUE TITLE
Xcode project doesn't compile any more when exporting from Unity 2018.2

### ISSUE THREAD

**Issue by pauberson** [jiulongw/swift-unity#95]
I added 2018.2 to the XcodePostBuild.cs but when trying to build the Xcode project I get this error:

/demo/xcode/DemoApp/Unity/Classes/Unity/DeviceSettings.mm:255:1: Control may reach end of non-void function

**Comment by MV-EricDeHaan** [comment]
That is a correct message from the compiler. Just add a `return deviceUnkown;` at line 255.
But then I receive 95 errors all with "Vuforia::" in the message...

**Comment by pauberson** [comment]
Yes, I then tried that and got a load of other errors too. This code is the same as in the normal unity build which does compile, so maybe there are some different compiler/xcode settings needed for 2018.2?

**Comment by ericqqqqq** [comment]
try to import Vuforia.framework for your project. 
read this, guys!
https://github.com/jiulongw/swift-unity/issues/89

**Comment by ericqqqqq** [comment]
also, instead of add return for non-void function, you guys can try to set Mismatched Return Type to No

**Comment by noahsark769** [comment]
The correct solution here is to set Mismatched Return Type to No in build settings.
